lang/Makefile

```diff
@@ -44,3 +44,9 @@ else
 	goyacc parser.y
 endif
 	@ROOT="$$( cd "$$( dirname "$${BASH_SOURCE[0]}" )" && cd .. && pwd )" && $$ROOT/misc/header.sh 'y.go'
+
+fuzz:
+	mkdir -p corpus
+	find .. -name \*.mcl -exec cp {} corpus \;
+	go-fuzz-build
```

Comment:
Should we add this dependency to the misc/make-deps.sh script? If anything gated by an --optional flag?